src/schema/scholarship/data.py

```diff
@@ -165,49 +159,36 @@ def get_cleansed_grade(grade_str):
     grade = re.findall(r" [ABC]+?0?-?", grade_str)
     total_grade = []
 
-    if not point == None:
+    if point is not None:
```

Comment:
파이썬은 none이 if문에 걸리면 false고 none이 아니면 true 라서 
```suggestion
    if point:
```
이렇게 고치면 될 것 같아 !